src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,72 @@
 import React from 'react';
 import './CurrentUser.scss';
+import PropTypes from 'prop-types';
+import { getUsers } from '../../api/api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: '',
+  };
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    this.userInfo();
+  }
+
+  componentDidUpdate(prevState) {
+    if (prevState.userId !== this.props.userId) {
+      this.userInfo();
+    }
+  }
+
+  userInfo = () => {
```

Comment:
```suggestion
  fetchUser = () => {
```
функцию мы обычно называем глаголом